workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectComponentDispatcherBase.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Improbable.Gdk.GameObjectRepresentation
 {
+    /// <summary>
+    ///     Base class for a GameObjectComponentDispatcher. Used in code generation.
```

Comment:
"Used in code generation" may not be relevant to users.

Reply:
Keep in mind in code docs are also useful for people writing code in the GDK Core

Reply:
This looks like the class should be internal